modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/objects/fields/Picklist.tsx

```diff
@@ -0,0 +1,162 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayForm, {ClaySelectWithOption} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import classnames from 'classnames';
+import React, {useCallback, useEffect, useState} from 'react';
+
+// @ts-ignore
+
+import {getListTypeEntries} from '../../data/listType';
+import ErrorMessage from '../ErrorMessage';
+import {TGenericFieldProps} from '../FieldsWrapper';
+
+const _valueToSelection = (
+	value: string | undefined,
+	options: TPicklistOption[]
+): TPicklistOption | undefined =>
+	options.find((option) => value && value === option.key);
+
+const DEFAULT_OPTION: TPicklistOption = {
```

Comment:
In MultiPickList we have the constants before the "private methods". Can we align this also here?

Reply:
Done, fixed all of the other SF issues meanwhile.